SwrveComponents/SwrveNode.xml

```diff
@@ -1,28 +1,57 @@
-<?xml version="1.0" encoding="utf-8" ?> 
-<component name="SwrveNode" extends="Group" xsi:noNamespaceSchemaLocation="https://devtools.web.roku.com/schema/RokuSceneGraph.xsd" >
-	   
+<?xml version="1.0" encoding="utf-8"?>
+<component name="SwrveNode" extends="Group">
+
 	<interface>
-        <field id="startSwrve" type="boolean" alwaysNotify="true"/>
-    </interface>
-    
+		<field id="configuration" type="assocarray" />
+		<field id="startSwrve" type="boolean" alwaysNotify="true" />
+
+		<field id="startSeconds" type="integer" alwaysNotify="false" />
```

Comment:
Globals have been moved to the `SwrveNode` and most have had their `Swrve` name space removed since the node it self is the name space.